tests/perf-dashboard/python_clientlibs_download.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
What do you think about moving putting putting the `perf-dashboard` directly at the top level, since it's not a test?  Or perhaps we can put both perf-dashboard and benchmarks into a 'metrics' directory?